optaplanner-core/src/main/java/org/optaplanner/core/config/partitionedsearch/PartitionedSearchPhaseConfig.java

```diff
@@ -45,7 +45,6 @@
 public class PartitionedSearchPhaseConfig extends PhaseConfig<PartitionedSearchPhaseConfig> {
 
     public static final String XML_ELEMENT_NAME = "partitionedSearch";
-    public static final String ACTIVE_THREAD_COUNT_AUTO = "AUTO";
     public static final String ACTIVE_THREAD_COUNT_UNLIMITED = "UNLIMITED";
```

Comment:
This is why I tend to prefer option 2, as sometimes there are extra values, like UNLIMITED.

Reply:
(But if we find good consistent naming, I don't veto option 1 though.)